api/v1alpha1/envoyproxy_types.go

```diff
@@ -73,6 +79,20 @@ type EnvoyProxySpec struct {
 	MergeGateways *bool `json:"mergeGateways,omitempty"`
 }
 
+// ProxyPorts defines the ports for the Proxy admin interface and so on.
+type ProxyPorts struct {
+	// AdminPort defines the port for the Proxy admin interface.
+	// If unspecified, defaults to 19000.
+	// +optional
+	AdminPort *int32 `json:"adminPort,omitempty"`
+
+	// ExtraPort defines the extra port for the Proxy extra interface like
+	// HealthCheck, Prometheus metrics.
+	// If unspecified, defaults to 19001.
+	// +optional
+	ExtraPort *int32 `json:"extraPort,omitempty"`
```

Comment:
`extraPort` is not a great name imo
should we split this up into `prometheusPort` & `healthCheckPort` ?

Reply:
nop, it will/should shared one port in most cases.

Reply:
can we then come up with a better name

Reply:
do you have any good idea?